icu4j/tools/build/src/main/java/com/ibm/icu/dev/tool/errorprone/ParseMavenOutForErrorProne.java

```diff
@@ -0,0 +1,109 @@
+//·©·2025·and·later:·Unicode,·Inc.·and·others.
+
+package com.ibm.icu.dev.tool.errorprone;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class ParseMavenOutForErrorProne {
+    private static final String RE_ERROR_PRONE_START =
+            "^\\[WARNING\\] ((?:[A-F]:)?[\\\\/a-zA-Z0-9_.\\-]+\\.java):\\[(\\d+),(\\d+)\\]"
+                + " \\[(\\S+)\\] (.+)";
+    private static final Pattern pattern = Pattern.compile(RE_ERROR_PRONE_START);
```

Comment:
as a constant, `pattern` should be uppercase, right?

Reply:
Done